src/app/modules/group/pages/group-overview/group-overview.component.html

```diff
@@ -7,4 +7,12 @@
   <alg-section icon="fa fa-chart-line" i18n-label label="Progress" *ngIf="group.currentUserCanWatchMembers">
     <alg-group-log-view [groupId]="group.id"></alg-group-log-view>
   </alg-section>
+
+  <alg-section icon="fa fa-sign-out-alt" i18n-label label="Leave the group ?" *ngIf="group.currentUserMembership !== 'none'">
+    <alg-group-leave
+      [groupId]="group.id"
+      [locked]="group.isMembershipLocked"
```

Comment:
Is there a reason not to pass one input `group` ?

Reply:
Fixed